### PR TITLE
Description for GOCDB roles

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR adds the description of the GOCDB roles in Service Provider guide of Check-in

### CODE REVIEW

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1146,6 +1168,37 @@ urn:mace:egi.eu:<GROUP-AUTHORITY>:[<GROUP>[:<SUBGROUP>:…]]:<ROLE>@<VO>
 urn:mace:egi.eu:aai.egi.eu:vm_operator@fedcloud.egi.eu
 ```
 
+### GOCDB Roles
+
+#### Background
+
+VO/group membership and role information is encoded in entitlements
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1175]: "GOCDB roles are encoded ..."

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -1090,6 +1111,7 @@ The following information about the authenticated user can be provided by EGI
 Check-in in order to control user access to resources:
 
 1. VO/group membership and role information about the authenticated user
+1. GOCDB roles
 1. Level of Assurance (LoA)
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1115]: 1. VO/group membership and role information
1. Capabilities
1. Identity assurance
1. GOCDB roles

File: content/en/providers/check-in/sp/_index.md [hunk 3]
```diff
@@ -223,6 +223,8 @@ attributes that are relevant for user authorisation:
 | Description                                                                                     | SAML Attribute         |
 | ----------------------------------------------------------------------------------------------- | ---------------------- |
 | [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information) | `eduPersonEntitlement` |
+| [Capabilities](#capabilities)                                                                   | `eduPersonEntitlement` |
+| [GOCDB roles](#gocdb-roles)                                                                     | `eduPersonEntitlement` |
 | [Level of Assurance (LoA)](#level-of-assurance)                                                 | `eduPersonAssurance`   |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:228]: "Level of Assurance" --> Identity Assurance

File: content/en/providers/check-in/sp/_index.md [hunk 4]
```diff
@@ -671,7 +673,9 @@ user authorisation:
 | Description                                                                                     | OIDC Claim              |
 | ----------------------------------------------------------------------------------------------- | ----------------------- |
 | [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information) | `eduperson_entitlement` |
-| [Level of Assurance (LoA)](#level-of-assurance)                                                 | `acr`                   |
+| [Capabilities](#capabilities)                                                                   | `eduperson_entitlement` |
+| [GOCDB roles](#gocdb-roles)                                                                     | `eduperson_entitlement` |
+| [Level of Assurance (LoA)](#level-of-assurance)                                                 | `eduperson_assurance`   |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:678]: Level of Assurance --> Identity Assurance

File: content/en/providers/check-in/sp/_index.md [hunk 5]
```diff
@@ -1021,6 +1025,48 @@ connected to Check-in.
 
 <!-- markdownlint-enable line-length no-inline-html -->
 
+#### 10.1 Capabilities
+
+<!-- markdownlint-disable line-length no-inline-html -->
+
+|          attribute name | Capabilities                                                                        |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | This attribute describes the resource or child-resource a user is allowed to access |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1034]: .. to access, optionally specifying certain actions the user is entitled to perform, as described in AARC-G027[https://aarc-community.org/guidelines/aarc-g027/]

File: content/en/providers/check-in/sp/_index.md [hunk 6]
```diff
@@ -1021,6 +1025,48 @@ connected to Check-in.
...
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.5923.1.1.1.7` (eduPersonEntitlement)                           |
+|          **OIDC scope** | `eduperson_entitlement`                                                             |
+|       **OIDC claim(s)** | `eduperson_entitlement`                                                             |
+| **OIDC claim location** | <ul><li>Userinfo endpoint</li><li>Introspection endpoint</li></ul>                  |
+|              **origin** | Capabilities are managed in the Community AAI                                       |
+|             **changes** | Yes                                                                                 |
+|        **multiplicity** | Multi-valued                                                                        |
+|        **availability** | Not always                                                                          |
+|             **example** | _urn:geant:helmholtz.de:res:nextcloud_at_kit#www.myIDP.de_                          |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1043]: replace example with "urn:mace:egi.eu:res:rcauth#aai.egi.eu"

File: content/en/providers/check-in/sp/_index.md [hunk 7]
```diff
@@ -1021,6 +1025,48 @@ connected to Check-in.
...
+<!-- markdownlint-disable line-length no-inline-html -->
+
+|          attribute name | Capabilities                                                                        |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | This attribute describes the resource or child-resource a user is allowed to access |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.5923.1.1.1.7` (eduPersonEntitlement)                           |
+|          **OIDC scope** | `eduperson_entitlement`                                                             |
+|       **OIDC claim(s)** | `eduperson_entitlement`                                                             |
+| **OIDC claim location** | <ul><li>Userinfo endpoint</li><li>Introspection endpoint</li></ul>                  |
+|              **origin** | Capabilities are managed in the Community AAI                                       |
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1039]: Capabilities are managed by Check-in

File: content/en/providers/check-in/sp/_index.md [hunk 8]
```diff
@@ -1090,7 +1136,9 @@ The following information about the authenticated user can be provided by EGI
 Check-in in order to control user access to resources:
 
 1. VO/group membership and role information about the authenticated user
+1. Capabilities
 1. Level of Assurance (LoA)
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1140]: Level of Assurance (LoA) --> Identity Assurance

File: content/en/providers/check-in/sp/_index.md [hunk 9]
```diff
@@ -1146,6 +1194,45 @@ urn:mace:egi.eu:<GROUP-AUTHORITY>:[<GROUP>[:<SUBGROUP>:…]]:<ROLE>@<VO>
...
 
+### Capabilities
+
+#### Background
+
+The user's capability information is encoded in entitlements
+(`eduPersonEntitlement` attribute values in SAML or `eduperson_entitlement`
+claim in OIDC). These entitlements are typically used to indicate access rights
+to protected resources. Entitlements are multi-valued, with each value formatted
+as a URN.
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1205]: as a URN following the syntax defined in AARC-G027[https://aarc-community.org/guidelines/aarc-g027/]

File: content/en/providers/check-in/sp/_index.md [hunk 10]
```diff
@@ -1177,3 +1264,34 @@ the selected authentication mechanism as follows:
...
+
+```vim
+urn:mace:egi.eu:goc.egi.eu:<PRIMARY_KEY>:<ON_ENTITY>:<USER_ROLE>@egi.eu
+```
+
+where:
+
+- `<PRIMARY_KEY>` is ...
+- `<ON_ENTITY>` is ...
+- `<USER_ROLE>` is ...
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1291]: is the user's role, e.g. "Site Operations Manager"

File: content/en/providers/check-in/sp/_index.md [hunk 11]
```diff
@@ -1177,3 +1264,34 @@ the selected authentication mechanism as follows:
...
+An entitlement value expressing GOCDB roles has the following syntax
+(components enclosed in square brackets are OPTIONAL):
+
+```vim
+urn:mace:egi.eu:goc.egi.eu:<PRIMARY_KEY>:<ON_ENTITY>:<USER_ROLE>@egi.eu
+```
+
+where:
+
+- `<PRIMARY_KEY>` is ...
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1289]: is the primary key for the user role, e.g. "123G0"

File: content/en/providers/check-in/sp/_index.md [hunk 12]
```diff
@@ -1177,3 +1264,34 @@ the selected authentication mechanism as follows:
...
+- `<ON_ENTITY>` is ...
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1290]: is the name of the entity on which the user role applies to, e.g. "GRIDOPS-GOCDB"

File: content/en/providers/check-in/sp/_index.md [hunk 13]
```diff
@@ -1177,3 +1264,34 @@ the selected authentication mechanism as follows:
 
 - Username/password credentials → Low
 - X.509 certification → Substantial
+
+### GOCDB Roles
+
+#### Background
+
+GOCDB roles are encoded in entitlements
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1272]: GOCDB roles, as per [GOCDB documentations](https://wiki.egi.eu/wiki/GOCDB/PI/get_user_method), are encoded

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: content/en/providers/check-in/sp/_index.md (hunk 3 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:228]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 8 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1140]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 4 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:678]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 5 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1034]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 6 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1043]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 7 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1039]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 9 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1205]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 13 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1272]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 11 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1289]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 12 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1290]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 10 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md:1291]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Comment by nikosev** [comment]
Closing this Draft PR and opening a new one in upstream repo [PR#206](https://github.com/EGI-Federation/documentation/pull/206)

**Pull request closed without merging**